cadquery/cq.py

```diff
@@ -3800,6 +3800,62 @@ def sphere(
         else:
             return self.union(spheres, clean=clean)
 
+    def cone(
+        self: T,
+        height: float,
+        radius: float = 0,
```

Comment:
Please remove the redundancy (at most two radius params needed) and defaulting of radius/radius1 :

```
        radius1: float,
        radius2: Optional[float] =None
```